### PR TITLE
General refactor, improvements for steps and common steps added

### PR DESCRIPTION
**Pull request by wtrocki**
## Motivation

Improve steps definitions in order to simplify documentation.
Add additional steps that may be useful. 

## Progress
- [x] Cleanup definitions and directives
- [x] Cleanup existing steps
- [x] Signature step integration

We going to provide other steps as part of the separate PR's

### CODE REVIEW

**Comment by wtrocki** [comment]
@witmicko @austincunningham - Worth checking this and reviewing to get famillar how steps can be created.

File: steps/step-signature/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+### Displaying signatures
+The `src` attribute of the image tag is bound to the `value` attribute of the signature tag.  To display a base64 encoded string, prefix it with the string: `data:image/png;base64,`
+```html
+<signature value="ctrl.model"></signature>
+```
+
+### Signature CSS styles
+Include the module SASS file in your application sass:
+```sass
+@import 'node_modules/fh-wfm-appform/sass/appform.scss';
```
Comment (JameelB) [inline steps/step-signature/README.md]: should be `@import 'node_modules/@raincatcher/step-signature/lib/signature.scss'`?

File: steps/step-signature/lib/angular/template/signature-form.tpl.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<md-subheader>Provide signature bellow</md-subheader>
```
Comment (JameelB) [inline steps/step-signature/lib/angular/template/signature-form.tpl.html]: `below`

File: .travis.yml [hunk 3]
```diff
@@ -17,6 +17,6 @@ cache:
     - packages/workflow-angular/node_modules
     - packages/workorder-angular/node_modules
     - packages/sync/node_modules
-    - examples/step-vehicle-inspection/node_modules
-    - examples/step-accident/node_modules
+    - steps/examples/step-vehicle-inspection/node_modules
+    - steps/examples/step-accident/node_modules
```
Comment (JameelB) [inline .travis.yml]: step-signature module does not need to be added here?

Comment (wtrocki) [reply .travis.yml]: Thanks for reminder! Adding!

File: steps/step-signature/lib/angular/template/signature-form.tpl.html (hunk 2 above)
Comment (wtrocki) [reply steps/step-signature/lib/angular/template/signature-form.tpl.html]: Actually I think it will be better with just `Provide Signature`.

**Review by witmicko — COMMENTED** [review]
looks good in general.
but
image

**Review by paolobueno — DISMISSED** [review]

**Review by JameelB — APPROVED** [review]
Verified Locally

**Pull request merged**